physics/equipotential_body.hpp

```diff
@@ -73,11 +73,18 @@ auto Equipotential<InertialFrame, Frame>::ComputeLine(
 
   Line equipotential;
   typename AdaptiveStepSizeIntegrator<ODE>::AppendState const append_state =
-      [&equipotential](State const& state) {
-        DependentVariables dependent_variables;
-        std::get<0>(dependent_variables) = std::get<0>(state.y).value;
-        std::get<1>(dependent_variables) = std::get<1>(state.y).value;
-        equipotential.push_back(dependent_variables);
+      [this, &equipotential, binormal, position, t](State const& state) {
+        auto const& [double_q, double_β] = state.y;
+        DependentVariables values;
+        auto& [q, β] = values;
+        q = double_q.value;
+        β = double_β.value;
+        DependentVariableDerivatives derivatives;
+        RightHandSide(
+            binormal, position, t, state.s.value, values, derivatives);
+        auto const& [qʹ, βʹ] = derivatives;
+        equipotential.Append(Instant() + state.s.value * (1 * Second),
```

Comment:
Let's have a constant for this `1 * Second`, if only to make it easier to locate the places where we have a cheezy conversion to times/speeds.  (Also on the next line.)